src/core/types/base/botConfig.ts

```diff
@@ -34,6 +34,10 @@ export interface BotConfig {
 	slackChannel?: string | undefined;
 	// Discord webhook url
 	discordWebhookUrl?: string | undefined;
+	// Telegram bot token
```

Comment:
I think now is the time we introduce a nested external logging config. Like we do with the skip setup. Since all of the external loggers are optional and defaults to console